Aquarius/Src/Aquarius/Events/Callbacks.h

```diff
@@ -0,0 +1,135 @@
+#pragma once
+
+#include "Aquarius/Core/Log.h"
+
+#include <GLFW/glfw3.h>
+
+
+namespace Aquarius {
+
+    class KeyboardEvent
+    {
+    public:
+        static void Callback(GLFWwindow* window, int key, int scanCode, int action, int modifications)
+        {
+            switch (action)
+            {
+                case GLFW_PRESS:
+                    AQ_INFO("key %v is pressed.", key);
+                    break;
+
+                case GLFW_RELEASE:
+                    AQ_INFO("key %v has been released.", key);
+                    break;
+
+                case GLFW_REPEAT:
+                    AQ_INFO("key %v is being pressed repeatedly.", key);
+                    break;
+
+                default:
+                    break;
+            }
+        }
+    };
+
+    class MouseEvent
```

Comment:
I think we should update these names to be MouseCallback or something. I do like defining them here and using them in the window class to register the callbacks